ui/packages/consul-ui/app/styles/layout.scss

```diff
@@ -49,7 +49,8 @@ html[data-route$='edit'] .app-view > header + div > *:first-child {
 /* needs a top margun :S */
 %app-view-content .tab-section > *:first-child:not(.filter-bar):not(table),
 %app-view-content .tab-section > .search-bar + p,
-%app-view-content .tab-section .consul-health-check-list {
+%app-view-content .tab-section .consul-health-check-list,
+%app-view-content .container {
```

Comment:
The Routing Config template does not have a `tab-nav`. I opted for using a generic class name to wrap around `Consul::DiscoveryChain` instead of using `tab-section` which wouldn't make sense.